custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/r_predictor.py

```diff
@@ -104,3 +104,68 @@ def predict(self, input_filename):
                 logger.error(error_message)
                 raise DrumCommonException(error_message)
         return py_data_object
+
+    def predict_unstructured(self, data, **kwargs):
+        def _r_is_character(r_val):
+            _is_character = ro.r("is.character")
+            return bool(_is_character(r_val))
+
+        def _r_is_raw(r_val):
```

Comment:
we dont need to run R interpreteter if value is bytes or string, its returned back as a typed rpy2 proxy, like StrVector or ByteVector, we check pythonic type of the proxy instead of running interpreter..

Reply:
added toTODO